### PR TITLE
Index get test

### PR DESCRIPTION
**Pull request by ruv-prog-so**

### CODE REVIEW

**Comment by RickyV33** [comment]
This looks fine to me. Not seeing anything that needs to be changed

File: test/indexGet.js [hunk 1]
```diff
@@ -3,17 +3,39 @@
 let chai = require('chai');
 let chaiHttp = require('chai-http');
 let server = require('../app');
```
Comment (chances) [inline test/indexGet.js]: Move this to below the third party requires with a blank line of padding.

File: test/indexGet.js [hunk 2]
```diff
@@ -3,17 +3,39 @@
...
-let should = chai.should();
+let expect = chai.expect;
+// For Rendering the html using ejs
```
Comment (chances) [inline test/indexGet.js]: Remove this comment; it's redundant with the comment on line 29

**Comment by chances** [comment]
LGTM

**Pull request merged**